pkg/tools/spire/start.go

```diff
@@ -60,55 +60,58 @@ func logrusEntry(ctx context.Context) *logrus.Entry {
 
 // AddEntry - adds an entry to the spire server for parentID, spiffeID, and selector
 //            parentID is usually the same as the agentID provided to Start()
-func AddEntry(ctx context.Context, parentID, spiffeID, selector string) error {
-	cmdStr := "spire-server entry create -parentID %s -spiffeID %s -selector %s -socketPath %s/api.sock"
-	cmdStr = fmt.Sprintf(cmdStr, parentID, spiffeID, selector, spireRoot)
+func AddEntry(ctx context.Context, parentID, spiffeID, selector, federatesWith string) error {
```

Comment:
Is it using by docker tests in current apps?

Reply:
Yes it is used